dbt/adapters/snowflake/impl.py

```diff
@@ -176,8 +176,8 @@ def timestamp_add_sql(self, add_to: str, number: int = 1, interval: str = "hour"
 
     @available.parse_none
     def submit_python_job(self, parsed_model: dict, compiled_code: str):
-        schema = getattr(parsed_model, "schema", self.config.credentials.schema)
-        database = getattr(parsed_model, "database", self.config.credentials.database)
+        schema = dict.get(parsed_model, "schema", self.config.credentials.schema)
```

Comment:
Should we figure out a way to test it? probably extract into a separate function?